plugins/dotenv/dotenv.plugin.zsh

```diff
@@ -1,14 +1,26 @@
 source_env() {
   if [[ -f $ZSH_DOTENV_FILE ]]; then
     if [ "$ZSH_DOTENV_PROMPT" != "false" ]; then
-      # confirm before sourcing file
-      local confirmation
-      # print same-line prompt and output newline character if necessary
-      echo -n "dotenv: source '$ZSH_DOTENV_FILE' file in the directory? (Y/n) "
-      read -k 1 confirmation; [[ "$confirmation" != $'\n' ]] && echo
-      # only bail out if confirmation character is n
-      if [[ "$confirmation" = [nN] ]]; then
-        return
+      # agree list
+      touch "${ZSH_DOTENV_AGREE_LIST_FILE}"
+      local dotenv_absolutepath="$(readlink -f $ZSH_DOTENV_FILE)"
```

Comment:
You can use `A` modifier to turn a file name into an absolute path and resolve symbolic link.
```suggestion
      local dotenv_absolutepath="${ZSH_DOTENV_FILE:A}"
```

Reply:
Thanks @Aloxaf !